regcore/db/django_models.py

```diff
@@ -1,46 +1,86 @@
 """Each of the data structures relevant to the API (regulations, notices,
 etc.), implemented using Django models"""
+import collections
+
 from django.core.exceptions import ObjectDoesNotExist
 
 from regcore.models import Diff, Layer, Notice, Regulation
 
 
+def get_tree(regs):
```

Comment:
Minor naming thing, but "tree" is a very overloaded term in the regs code. It looks like you're creating a map of child nodes by parent label, instead?